File: guides/java/kora-java-guide-getting-started-app/src/test/java/ru/tinkoff/kora/guide/gettingstarted/GettingStartedAppTest.java
package ru.tinkoff.kora.guide.gettingstarted;

import static org.junit.jupiter.api.Assertions.assertNotNull;

import org.junit.jupiter.api.Test;
import ru.tinkoff.kora.test.extension.junit5.KoraAppTest;
import ru.tinkoff.kora.test.extension.junit5.TestComponent;

@KoraAppTest(Application.class)
class GettingStartedAppTest {

    @TestComponent
    private HelloController helloController;

    @Test
    void controllerIsWired() {
        assertNotNull(helloController);
    }
}

File: guides/kotlin/kora-kotlin-guide-getting-started-app/src/test/kotlin/ru/tinkoff/kora/guide/gettingstarted/GettingStartedAppTest.kt
package ru.tinkoff.kora.guide.gettingstarted

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Assertions.assertNotNull
import org.junit.jupiter.api.Test
import ru.tinkoff.kora.test.extension.junit5.KoraAppTest
import ru.tinkoff.kora.test.extension.junit5.TestComponent

@KoraAppTest(Application::class)
class GettingStartedAppTest {
    @TestComponent
    lateinit var helloController: HelloController

    @Test
    fun componentsAreWired() {
        assertNotNull(helloController)
    }

    @Test
    fun helloControllerReturnsPlainTextResponse() {
        val response = helloController.hello()
        assertEquals(200, response.code())
    }
}
